server/paymentProviders/transferwise/index.ts

```diff
@@ -67,7 +68,57 @@ async function payExpense(connectedAccount, payoutMethod, expense): Promise<any>
   return { quote, recipient, transfer, fund };
 }
 
+async function getRequiredBankInformation(collective, currency: string): Promise<any> {
+  const cacheKey = `transferwise_required_bank_info_${collective.id}_to_${currency}`;
```

Comment:
Good idea to cache this :+1: 

Maybe we could cache at the host level, because most of the time collectives have the same currency than the host and for OSC which has 1700+ collectives this can make a difference.
```suggestion
  const cacheKey = `transferwise_required_bank_info_${hostCollective.id}_${collective.currency}_to_${currency}`;
```

Reply:
Nice catch! I refactored the way we move things around and I'm now caching on the host level! Thank you!